scss/styles.scss

```diff
@@ -0,0 +1,78 @@
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.main-title {
+  font-size: $font-big;
+  position: relative;
+  margin-bottom: 100px;
+  padding-left: 100px;
+  padding-top: 100px;
+
+  &::before {
+    background-repeat: no-repeat;
+    background-image: url('resources/triangle.svg');
+    content: '';
+    height: 100px;
+    left: 45px;
+    position: absolute;
+    top: 50px;
+    width: 100px;
+    z-index: -1;
+  }
+}
+
+.tech-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  padding-left: 120px;
```

Comment:
I would set it left and right for symmetry. `padding: 0 120px;` (that says 0 in top and bottom and 120px in left and right)